app.mjs

```diff
@@ -534,9 +534,9 @@ const cmdLocation = async (_systemData, _userData, relay, ev) => {
   const location = ev.content.match(REGEX_LOCATION) ? ev.content.match(REGEX_LOCATION)[2] : ev.content.match(REGEX_LOCATION_ALT) ? ev.content.match(REGEX_LOCATION_ALT)[1] : "";
   let message = "わかりませんでした…";
   if (!!location) {
-    const geoDatas = await getLocation(location);
-    if (!!geoDatas.length) {
-      const geoData = geoDatas[0];
+    const getDataItems = await getLocation(location);
```

Comment:
I understand that "Datas" should be "DataItems", but why did you change geo to get?

Reply:
Wow, I'll fix soon